api/app/src/sequelize/migrations/2023050403101805-update-patient-data.ts

```diff
@@ -0,0 +1,70 @@
+import { DataTypes } from "sequelize";
+import type { Migration } from "..";
+import { PatientModel } from "../../models/medical/patient";
```

Comment:
I came to the realization we shouldn't  refer to anything from the app that change over time, as migrations might be executed when those files are on future versions, whereas this migration expects the code on a certain point in time.

I suggest we don't import and use the table name as it is now - as a pattern/practice.

Reply:
Yeah that's a good point, will keep this in mind going forward